modules/siddhi-core/src/main/java/io/siddhi/core/util/error/handler/util/ErrorType.java

```diff
@@ -23,5 +23,6 @@
  */
 public enum ErrorType {
     MAPPING,
-    TRANSPORT
+    TRANSPORT,
+    STORE
```

Comment:
We have used `TRANSPORT` for errors occurring due to unavailability of an endpoint in a Sink.
Since unavailability of DB connection also goes close by, shall we use `TRANSPORT` instead of `STORE` here? 
WDYT?

Reply:
Now I think about it, I feel it is better to keep this distiction. If we add several types of transport errors in the future, then we need bulk if conditions to determine the subsequent code flow.

Reply:
Removed STORE type in https://github.com/siddhi-io/siddhi/pull/1682/commits/cb5d94b2ec47b0dea275f32c13dac81ade0a6698